packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -110,13 +118,17 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 	}
 
 	setSoftDeletionInterval() {
-		this.logger.debug('Setting soft-deletion interval (pruning) for executions');
+		this.logger.debug(
+			`Setting soft-deletion interval (pruning) for executions with rate ${this.rates.hardDeletion}`,
+		);
 
 		this.intervals.softDeletion = setInterval(async () => this.prune(), this.rates.hardDeletion);
```

Comment:
I noticed we are using `hardDeletion` rate even for the soft-deletes. Is this intentional or should be it changed?

Reply:
Good catch, if you can please update.

Reply:
Fixed. Btw, is it intentional that hard-deletion happens every 15min but soft-deletion only every 1h?